ext/xmlreader/tests/XML_variant_static_call.phpt

```diff
@@ -0,0 +1,18 @@
+--TEST--
+XMLReader: XML variant static call
+--SKIPIF--
+<?php if (!extension_loaded("xmlreader")) print "skip XMLReader extension required"; ?>
+--FILE--
+<?php
+
+$xml = '<?xml version="1.0" encoding="UTF-8"?><books></books>';
+
+$reader = XMLReader::XML($xml);
+while ($reader->read()) {
+  echo $reader->name."\n";
+}
+?>
+--EXPECTF--
+Deprecated: Non-static method XMLReader::XML() should not be called statically in %s on line %d
```

Comment:
May we also remove this one? Deprecated features will be removed someday, so let's add no tests to difficult that :)